app/controllers/catalog_controller.rb

```diff
@@ -405,7 +416,8 @@ class CatalogController < ApplicationController
     config.add_show_field 'other_format_display', label: 'Other format(s)'
     config.add_show_field 'indexes_display', label: 'Indexes'
     config.add_show_field 'finding_aid_display', label: 'Finding aid'
-    config.add_show_field 'cumulative_index_finding_aid_display', label: "Cumulative index/\u200BFinding aid"
+    config.add_show_field 'cumulative_index_finding_aid_display', label: 'Cumulative index/\u200BFinding aid'
```

Comment:
```suggestion
    config.add_show_field 'cumulative_index_finding_aid_display', label: "Cumulative index/\u200BFinding aid"
```

Shouldn't we keep this with double quotes, so it doesn't display `\u200B` literally?